core_impl/src/main/java/io/opencensus/impl/tags/TagContextSerializerImpl.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017, Google Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opencensus.impl.tags;
+
+import io.opencensus.tags.TagContext;
+import io.opencensus.tags.TagContextSerializer;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+final class TagContextSerializerImpl extends TagContextSerializer {
```

Comment:
Is there a specific reason to keep `SerializationUtils` separated from this class?